node/fs.js

```diff
@@ -77,13 +88,22 @@ function gunzip(bufferOrString, options) {
   options = typeof options === "string" ? { encoding: options } : options;
   options = Object.assign(Object.create(null), options);
 
-  const stream = new zlib.Gunzip(options);
-  if (options.encoding !== "utf8") {
-    return streamToBuffer(stream, bufferOrString);
+  if (useGunzipFastPath) {
+    try {
+      const stream = new minizlib.Gunzip(options);
+      if (options.encoding === "utf8") {
+        let result = "";
+        stream.on("data", chunk => result += chunk).end(bufferOrString);
+        return result;
```

Comment:
I suspect they'll fix the issue but until this having a fallback isn't a bad idea.